bp/index.html

```diff
@@ -4700,6 +4716,103 @@ <h2>Cross reference of use case requirements against best practices</h2>
+</li></ul>
+
+<h3>Reusable</h3>
+	    <p>"The ultimate goal of FAIR is to optimise the reuse of data. To achieve this, metadata and data should be well-described so that they can be replicated and/or combined in different settings.</p>
+<ul><li>
+<a href="https://www.go-fair.org/fair-principles/r1-metadata-richly-described-plurality-accurate-relevant-attributes/">R1. (Meta)data are richly described with a plurality of accurate and relevant attributes</a>
+<br/>
+See <a href="https://www.w3.org/TR/dwbp/#StructuralMetadata">DWBP Best Practice 3: Provide structural metadata</a>, which doesn’t require the attibution to be 'richly described' or for there to be many attributes (let alone that they be accurate & relevant); but it does at least say that whatever attribution you do provide should be described.
+<br/>
+See <a href="https://www.w3.org/TR/dwbp/#EnrichData">DWBP Best Practice 31: Enrich data by generating new data</a>
+</li>
+<li><a href="https://www.go-fair.org/fair-principles/r1-1-metadata-released-clear-accessible-data-usage-license/">R1.1. (Meta)data are released with a clear and accessible data usage license</a>
+<br/>
+<a href="https://www.w3.org/TR/dwbp/#DataLicense">DWBP Best Practice 4: Provide data license information</a>; again, this doesn’t say that the license should be "clear", but it does say that it should be accessible - "attached to data" (via a link from the metadata or embedded in the metadata).
+</li><li>
+<a href="https://www.go-fair.org/fair-principles/r1-2-metadata-associated-detailed-provenance/">R1.2. (Meta)data are associated with detailed provenance</a>
+<br/>
+<a href="https://www.w3.org/TR/dwbp/#DataProvenance">DWBP Best Practice 5: Provide data provenance information</a> "Provide complete information about the origins of the data and any changes you have made."
+</li><li>
+<a href="https://www.go-fair.org/fair-principles/r1-3-metadata-meet-domain-relevant-community-standards/">R1.3. (Meta)data meet domain-relevant community standards</a>
+<br/>
+See <a href="https://www.w3.org/TR/dwbp/#MachineReadableStandardizedFormat">DWBP Best Practice 12: Use machine-readable standardized data formats</a> "Make data available in a machine-readable, standardized data format that is well suited to its intended or potential use." This addresses the "low level", data format, aspect of (community) standards
+<br/>
+<a href="https://www.w3.org/TR/dwbp/#ReuseVocabularies">DWBP Best Practice 15: Reuse vocabularies, preferably standardized ones</a> "Use terms from shared vocabularies, preferably standardized ones, to encode data and metadata." This addresses more of the semantic standardization that exists in some community standards: "Use of vocabularies already in use by others captures and facilitates consensus in communities."
+
+	    <i>I’m not so sure of the match(es) here</i>
```

Comment:
Should this be removed?

Reply:
Otherwise make this an [`ednote`](https://github.com/w3c/respec/wiki/ednote)?